src/components/Card/EmployeeCardContainer.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import { useFormik } from 'formik';
+
+import { formValidation } from '../../validation/validation';
+import { useEditEmployee } from '../../api/editEmployee';
+import { useDeleteEmployee } from '../../api/deleteEmployee'
+
+import EmployeeCard from './EmployeeCard';
+
+const defaultValues = {
+  firstName: '',
+  lastName: '',
+  email: '',
+  age: 0,
+};
+
+const EmployeeCardContainer = ({ firstName, lastName, email, age, id }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const { mutate: editEmployeeMutation } = useEditEmployee();
+
+  const { mutateAsync: deleteEmployeeMutation } = useDeleteEmployee();
+
+  const handleDeleteEmployee = async () => {
+    await deleteEmployeeMutation(id);
+  };
+
+  const formik = useFormik({
+    initialValues: defaultValues,
+    onSubmit: (employeeFormData, onSubmitProps) => {
+      editEmployeeMutation({ ...employeeFormData, activeEmployeeId: id });
+      onSubmitProps.resetForm();
+      setIsOpen(false)
+    },
+    validationSchema: formValidation
+  });
+
+  useEffect(() => {
+    formik.setValues({ firstName: firstName, lastName: lastName, email: email, age: age });
+  }, [firstName, lastName, email, age]);
+
+  const handleCancel = () => {
+    formik.setValues({ firstName: firstName, lastName: lastName, email: email, age: age });
```

Comment:
Because if we close the modal window, but we don't want to save the changes, but just want to close modal, then when the modal window is reopened, data, entered earlier, will be displayed. 
Perhaps there is a better way to avoid such an error.